api/services/CaveService.js

```diff
@@ -161,10 +174,16 @@ module.exports = {
 
     if (!cave) return null;
 
-    [cave.massifs, cave.descriptions, cave.documents] = await Promise.all([
+    [
+      cave.massifs,
+      cave.descriptions,
+      cave.documents,
+      cave.exploringOrganizations,
```

Comment:
Instead of fetching `exploringOrganizations` here, adding a `.populate('entrances')` above with a `NameService.setNames()` should also work and it saves a request to the DB

Reply:
Got it.